src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -154,6 +156,30 @@ const DataSeriesFormItemModal = ({
   const { id, type } = cardConfig;
   const baseClassName = `${iotPrefix}--card-edit-form`;
 
+  const handleTranslation = useCallback(
+    (idToTranslate) => {
```

Comment:
can we align the use of these arguments. In CardEditFormContent.jsx we are using `id` and here we are using `idToTranslate`. I prefer the more explicit option.

Reply:
sure